template/template.go

```diff
@@ -74,30 +76,70 @@ func Named(s string) (*named, error) {
 	return nil, errors.New("no matching template found")
 }
 
+var DefaultTemplate, _ = Parse("{{ .Prompt }}")
```

Comment:
The new templating is powerful, but writing templates gets quite hard – it would be great to include a `template.md` document in `docs` that outlines how to write one. I don't imagine most Ollama users will have to, but we should still try and make it as approachable as possible.

Reply:
that can be a follow up once this is merged